broker/cloud_run/lsst/variability/main.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""This module produces "value-added" lite alerts containing StetsonJ statistics on the DIA point source fluxes."""
+
+import os
+from typing import Dict
+import numpy as np
+import pandas as pd
+import flask
+import pittgoogle
+from google.cloud import logging
+
+# [FIXME] Make this helpful or else delete it.
+# Connect the python logger to the google cloud logger.
+# By default, this captures INFO level and above.
+# pittgoogle uses the python logger.
+# We don't currently use the python logger directly in this script, but we could.
+logging.Client().setup_logging()
+
+PROJECT_ID = os.getenv("GCP_PROJECT")
+TESTID = os.getenv("TESTID")
+SURVEY = os.getenv("SURVEY")
+
+# Variables for incoming data
+# A url route is used in setup.sh when the trigger subscription is created.
+# It is possible to define multiple routes in a single module and trigger them using different subscriptions.
+ROUTE_RUN = "/"  # HTTP route that will trigger run(). Must match deploy.sh
+
+# Variables for outgoing data
+HTTP_204 = 204  # HTTP code: Success
+HTTP_400 = 400  # HTTP code: Bad Request
+
+# GCP resources used in this module
+TOPIC = pittgoogle.Topic.from_cloud(
+    "value-added", survey=SURVEY, testid=TESTID, projectid=PROJECT_ID
+)
+
+app = flask.Flask(__name__)
+
+
+@app.route(ROUTE_RUN, methods=["POST"])
+def run():
+    """Produces a value-added alert stream (${survey}-value-added) containing StetsonJ statistics on the DIA point
+    source fluxes. Messages in this stream retain fields from the original alert-lite packet.
+
+    This module is intended to be deployed as a Cloud Run service. It will operate as an HTTP endpoint
+    triggered by Pub/Sub messages. This function will be called once for every message sent to this route.
+    It should accept the incoming HTTP request and return a response.
+
+    Returns
+    -------
+    response : tuple(str, int)
+        Tuple containing the response body (string) and HTTP status code (int). Flask will convert the
+        tuple into a proper HTTP response. Note that the response is a status message for the web server.
+    """
+    # extract the envelope from the request that triggered the endpoint
+    # this contains a single Pub/Sub message with the alert to be processed
+    envelope = flask.request.get_json()
+    try:
+        alert_lite = pittgoogle.Alert.from_cloud_run(envelope, schema_name="default")
+    except pittgoogle.exceptions.BadRequest as exc:
+        return str(exc), HTTP_400
+
+    TOPIC.publish(
+        pittgoogle.Alert.from_dict(
+            {**alert_lite.dict, "variability": _calculate_stetsonJ_statistics(alert_lite)},
```

Comment:
```suggestion
    stetsonj_stats = _calculate_stetsonJ_statistics(alert_lite)
    TOPIC.publish(
        pittgoogle.Alert.from_dict(
            {**alert_lite.dict, "variability": stetsonj_stats},
```

As a reader, it would help me to see the stats being defined on their own line, since that is a significant data transformation that this function makes.